### PR TITLE
Histogram Simulator Link added to the Home Page under Other DashBoards

### PR DESCRIPTION
**Pull request by ab0092**
@chutten changes can be viewed at https://ab0092.github.io/telemetry-dashboard/

### CODE REVIEW

File: new-pipeline/style/dashboards.css [hunk 1]
```diff
@@ -43,6 +43,13 @@
     z-index: 1;
     top: 5px;
 }
+
+/** Max Width Setting For e10s **/
+#filter-e10s + .btn-group ul
+{
+    max-width: 10vw !important;
+}
+
```
Comment (chutten) [inline new-pipeline/style/dashboards.css]: This code has already been reviewed and merged. Maybe rebase your working copy atop the latest from mozilla/telemetry-dashboard?

File: index.html [hunk 2]
```diff
@@ -105,6 +105,9 @@
                     <a class="list-group-item" href="https://squarewave.github.io">
                         Background Hang Reporting <div class="dashboard-description">View browser hang information collected from Nightly users</div>
                     </a>
+                    <a class="list-group-item" href="histogram-simulator/">
+                        Histogram Simulator <div class="dashboard-description">View Bucket Handling Information</div>
```
Comment (chutten) [inline index.html]: Fits in nicely with the rest, but the description should include what you can do there. "Simulate histogram bucket widths" would work

**Comment by ab0092** [comment]
I have added the appropriate description for the link, could you please review

**Review by chutten — APPROVED** [review]

**Comment by chutten** [comment]
Thank you for your help! Would you like to find your own next bug, or would you like some help?

**Comment by ab0092** [comment]
A little help from your end will be great as i am still trying to get
around what the telemetry dash-board is all about. Thanks.

**Comment by chutten** [comment]
Anything listed as "mentored" on telemetry-dashboard should be good starting places for bugs. But you've already coded one so maybe you're interested in something bigger? Maybe #257 ?

**Comment by ab0092** [comment]
Yes certainly i would love to work on some bigger issues, i was about to
ask you about the #257 issue, could you please provide me with some more
details about this issue.

**Comment by chutten** [comment]
Sure, let's take the conversation over there

**Pull request merged**